robolectric/src/test/java/org/robolectric/shadows/ShadowCameraDeviceImplTest.java

```diff
@@ -72,12 +71,28 @@ public void tearDown() throws CameraAccessException {
 
   @Test
   @Config(sdk = VERSION_CODES.P)
```

Comment:
Instead of having three separate methods you can just do this:
```java
@Config(minSdk=LOLLIPOP)
public void createCaptureRequest() throws CameraAccessException {
... 
}
```